src/emmy/abstract/function.cljc

```diff
@@ -311,3 +301,11 @@
               (f/arity f)
               (domain-types f)
               (range-type f)))
+
+(defmethod g/zero-like [::function] [^Function a] (fn [& _] (g/zero-like (.-range a))))
```

Comment:
does the range always have an instance of the output type? I thought we could have things like `'(UP Real Real)`.

I have a PR bringing in the Sussman-style here: https://github.com/mentat-collective/emmy/pull/113

I wonder if this implementation will work there, or what you think...